bot/start.ts

```diff
@@ -426,6 +427,8 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<MainCont
       order.status = 'COMPLETED_BY_ADMIN';
       const buyer = await User.findOne({ _id: order.buyer_id });
       const seller = await User.findOne({ _id: order.seller_id });
+      if (buyer === null) return;
+      if (seller === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program